JAGPropertyConverter/JAGPropertyConverter.m

```diff
@@ -506,7 +536,23 @@ - (void) setPropertiesOf: (id) object fromDictionary: (NSDictionary*) dictionary
             NSString *propertyName = self.enableSnakeCaseSupport ? [dictKey asCamelCaseFromUnderscore] : dictKey;
             value = [self composeModelFromObject: value withTargetClass:propertyClass propertyName:propertyName];
         }
-        if ([property canAcceptValue:value]) {
+
+        if (isKeyPath && ![remainingKeyPath isEqualToString:@""]) {
+            id ownedObject;
+
+            if ( ! [object valueForKey:key]) {
+                [object setValue:[[property.propertyClass alloc] init] forKey:key];
+            }
+
+            ownedObject = [object valueForKey:key];
+            isKeyPath = [remainingKeyPath rangeOfString:@"."].location != NSNotFound;
+
+            if (!isKeyPath) {
+                [ownedObject setValue:value forKey:remainingKeyPath];
```

Comment:
this is not entirely correct. you need to also check `canAcceptValue:` like the if below in line 555